upcoming_changes/72.new.rst

```diff
@@ -0,0 +1 @@
+Support for [Dirac GOS](https://zenodo.org/records/12800856) in the gosh format for EELS quantification, which includes the relativistic effects (e.g. spin-orbit coupling) based on Dirac equation. 
```

Comment:
This is a `rst` file and rst formatting needs to be used: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages
```suggestion
Support for `Dirac GOS <https://zenodo.org/records/12800856>`_ in the gosh format for EELS quantification, which includes the relativistic effects (e.g. spin-orbit coupling) based on Dirac equation. 
```